src/main/java/core/basesyntax/service/operationhandler/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.operationhandler;
+
+public interface OperationHandler {
+    int getValueByOperation(int value);

```

Comment:
You should do the operation in method
```suggestion
    int calculate(int currentAmount, int operationAmount);
```